### DISCUSSION TITLE
github.com/hbstack/blog/modules/post-nav/v0.3.8

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#780] [General]
# github.com/hbstack/blog/modules/post-nav/v0.3.8

0.3.8 (2024-05-01) Styles 🎨 post-nav: minor tweak (#807) (62b50d9) 

https://hbstack.dev/releases/github.com-hbstack-blog/modules-post-nav-v0.3.8/

**Comment by LITUATUI** [comment]
Much better now, thanks!

**Reply by razonyang** [reply]
Glad you like it.
